Project.toml

```diff
@@ -5,18 +5,19 @@ version = "0.1.3"
 
 [deps]
 CodecZlib = "944b1d66-785c-5afd-91f1-9de20f533193"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 MD5 = "6ac74813-4b46-53a4-afec-0b5dc9d7885c"
 Mixers = "2a8e4939-dab8-5edc-8f64-72a8776f13de"
 Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
```

Comment:
Is the resorting intentional or does it come from some IDE? 😉

Reply:
I did the `] activate .` then `add` thing

Reply:
The comment was targeting the reordering in `compat` (MD5.jl etc.), but never mind, I was just curious ;)

Reply:
yeah, I think the re-ordering was also done by `Pkg`, I'm on `1.4.1`

Reply:
Alright, then only the `using DataFrames` in `runtests.jl` is missing to pass the CI!